kin-ecosystem-sdk/src/main/java/com/kin/ecosystem/data/auth/AuthLocalData.java

```diff
@@ -120,15 +122,21 @@ public String getUserID() {
 		return signInSharedPreferences.getString(USER_ID_KEY, null);
 	}
 
+	@Override
+	public String getEcosystemUserID() {
+		return signInSharedPreferences.getString(ECOSYSTEM_USER_ID_KEY, null);
```

Comment:
default string  "N/A" or "null" maybe?

Reply:
for the local should return as null not empty string or something